extensions/ql-vscode/src/test-adapter.ts

```diff
@@ -221,10 +221,26 @@ export class QLTestAdapter extends DisposableObject implements TestAdapter {
       cancellationToken: cancellationToken,
       logger: testLogger
     })) {
+      const state = event.pass
+        ? 'passed'
+        : event.messages?.length
+        ? 'errored'
```

Comment:
Would `failed due to errors` or `failed to run` be more meaningful for the user? I had to think about the difference between 'errored' and 'failed' for a moment.

Reply:
That won't work. `state` is a property of the `TestEvent` interface and only allows `'running' | 'passed' | 'failed' | 'skipped' | 'errored'`. The hope is that the messages will be more explicit as to what happened.

Reply:
Oh fair enough. We can consider having more detail in the way `message` is constructed below, but no need to block this PR.